includes/views/html-admin-missing-woocommerce.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Missing WooCommerce notice.
+ *
+ * @package WC_Beta_Tester\Admin\Views
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+?>
+
+<div class="notice notice-error">
+	<p><strong><?php esc_html_e( 'WooCommerce Beta Tester', 'woocommerce-beta-tester' ); ?></strong> <?php esc_html_e( 'requires WooCommerce to be installed and enabled.', 'woocommerce-beta-tester' ); ?></p>
```

Comment:
I think these 2 strings need to be merged with a placeholder for plugin name, otherwise it won't be properly translatable.